foundry.toml

```diff
@@ -2,6 +2,7 @@
   evm_version = "paris"
```

Comment:
[arbitrary location for comment]

can you change the CI coverage threshold to 0 so that CI passes?

Reply:
Done 5f1cada9a5f454e7ed5267b4ba46b4820a9b9d70